spec/support/serializers.rb

```diff
@@ -113,6 +112,38 @@ def meta
     end
   end
 
+  class PostSerializerWithoutIncludeLinks
+    include JSONAPI::Serializer
+
+    attribute :title
+    attribute :long_content do
```

Comment:
Consider dropping the `long_content` attribute from both of these serializers and the tests, since we don't care about testing this behavior here (it's tested elsewhere) so it's kind of just noise and could be excluded.